src/stylesheets/_generic.scss

```diff
@@ -70,3 +70,8 @@ $generic-count: 10;
         -webkit-box-shadow: inset 0 0 10px $gray-light;
     }
 }
+
+code {
```

Comment:
This is too generic of a directive, let's remove bootstrap style instead

Reply:
We can't edit bootstrap, only override like this. 
We can't remove bootstrap easily, because ThreeJSEditor fullscreen capability seems to be dependent on that. The only place where bootstrap is in effect now.